app/src/main/java/openfoodfacts/github/scrachx/openfood/network/OpenFoodAPIClient.java

```diff
@@ -87,6 +87,7 @@
     private static OkHttpClient httpClient = Utils.HttpClientBuilder();
     private final OpenFoodAPIService apiService;
     private Context mActivity;
+    private final String fieldsToFetchProductDetail = "brands,product_name_fr,product_name,image_small_url,image_front_thumb_url,quantity,nutrition_grades_tags";
```

Comment:
It is a constant, so i would rename it to:
```suggestion
    private final String FIELDS_TO_FETCH_FACETS = "brands,product_name_fr,product_name,image_small_url,image_front_thumb_url,quantity,nutrition_grades_tags";
```

Reply:
Also what @teolemon said, `product_name_fr` should be `product_name_xx` with `xx` being the phone locale.